aten/src/THC/generic/THCTensorMathPointwise.cu

```diff
@@ -443,25 +370,8 @@ void THCTensor_(tpow)(THCState *state, THCTensor *self_, real value, THCTensor *
 THC_API void
 THCTensor_(cdiv)(THCState* state, THCTensor *self_, THCTensor *src1, THCTensor *src2)
 {
-  THCAssertSameGPU(THCTensor_(checkGPU)(state, 3, self_, src1, src2));
-  THArgCheck(THCTensor_(nElement)(state, src1) ==
-             THCTensor_(nElement)(state, src2), 3, "sizes do not match");
-
-  if (self_ == src1) {
-    // self /= src2
-    if (!THC_pointwiseApply2<real, real>(state, self_, src2, TensorDivOp<real>())) {
-      THArgCheck(false, 2, CUTORCH_DIM_WARNING);
-    }
-  } else {
-    THCTensor_(resizeAs)(state, self_, src1);
-
-    // self = src1 / src2
-    if (!THC_pointwiseApply3<real, real, real>(state, self_, src1, src2, TensorDivOp<real>())) {
-      THArgCheck(false, 2, CUTORCH_DIM_WARNING);
-    }
-  }
-
-  THCudaCheck(cudaGetLastError());
+  auto out = at::Tensor(self_, true);
+  at::div_out(out, at::Tensor(src1, true), at::Tensor(src2, true));
```

Comment:
This probably won't matter, but this function will broadcast now and didn't before (this applies to all the functions in this diff).  I guess it's possible that this could hide errors if callers expected these to match exactly, but this seems fine if no tests break.